python/tests/tsutil.py

```diff
@@ -242,30 +242,45 @@ def insert_random_ploidy_individuals(ts, max_ploidy=5, max_dimension=3, seed=1):
     return tables.tree_sequence()
 
 
-def insert_individuals(ts, samples=None, ploidy=1):
+def insert_individuals(ts, nodes=None, ploidy=1, allow_mixed_ploidy=False):
```

Comment:
Do we use this ``allow_mixed_ploidy`` argument? If not, why add it?

Reply:
Re deep_history - if we graft deep history onto the simulation we no longer have individuals for all the nodes (the new "deep history" nodes don't have individuals)

Reply:
Re mixed ploidy, sure, can remove. It just seemed like a useful thing to flag up to our future selves. Perhaps all that's needed is to make a comment/warning in the docstring that this utility function could result in mixed ploidies?

Reply:
sounds good